components/grid/__tests__/index.test.tsx

```diff
@@ -88,15 +83,6 @@ describe('Grid', () => {
     expect(asFragment().firstChild).toMatchSnapshot();
   });
 
-  it('ResponsiveObserve.unsubscribe should be called when unmounted', () => {
```

Comment:
Do not remove directly which should replace with new test case.

Reply:
I have no idea how to replace it. What do I do ? Who write the new test ?

Reply:
Do not remove existing test cases, add some new test cases for this feature directly.

Reply:
@yoyo837 @zombieJ 

I have no idea how to replace it. What do I do ? Who write the new test ?

Reply:
@yoyo837 @zombieJ 
Tell me at least, the behaviour you expect to be tested ? And I will try to think about the test implementation

Reply:
Let me handle this : )